Carrot-Assistant/components/embeddings.py

```diff
@@ -30,6 +42,22 @@ def __init__(
             model: EmbeddingModel,
             search_kwargs: dict,
             ) -> None:
+        """
+        Initialises the connection to an embeddings database
+
+        Parameters
+        ----------
+        embeddings_path: str
+            A path for the embeddings database. If one is not found, it will be built, which takes a long time. This is built from concepts fetched from the OMOP database
+        force_rebuild: bool
+            If true, the embeddings database will be rebuilt.
+        embed_vocab: List[str]
```

Comment:
can we change the name of this variable from embed_vocab to embed_vocabulary? I know Reza tends to cut the name of the variables, but I generally prefer longer names that facilitate code reading